docs/api/1.0/index.rst

```diff
@@ -0,0 +1,96 @@
+.. _kinto-api-endpoints:
```

Comment:
It seems to me that the docs are not versioned on each HTTP API increment, but on each Python package release, which is different.

So how can we have document with:

- a list of all API versions ?
- for each new API version a specific document with the diff with the previous API version